docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,160 @@
+insights                                   4.3.0     True        False         False      48m
+kube-apiserver                             4.3.0     True        False         False      45m
+kube-controller-manager                    4.3.0     True        False         False      45m
+kube-scheduler                             4.3.0     True        False         False      45m
+machine-api                                4.3.0     True        False         False      47m
+machine-config                             4.3.0     True        False         False      43m
+marketplace                                4.3.0     True        False         False      43m
+monitoring                                 4.3.0     True        False         False      40m
+network                                    4.3.0     True        False         False      48m
+node-tuning                                4.3.0     True        False         False      44m
+openshift-apiserver                        4.3.0     True        False         False      44m
+openshift-controller-manager               4.3.0     True        False         False      46m
+openshift-samples                          4.3.0     True        False         False      42m
+operator-lifecycle-manager                 4.3.0     True        False         False      47m
+operator-lifecycle-manager-catalog         4.3.0     True        False         False      47m
+operator-lifecycle-manager-packageserver   4.3.0     True        False         False      45m
+service-ca                                 4.3.0     True        False         False      48m
+service-catalog-apiserver                  4.3.0     True        False         False      44m
+service-catalog-controller-manager         4.3.0     True        False         False      44m
+storage                                    4.3.0     True        False         False      43m
+support                                    4.3.0     True        False         False      48m
+```
+
+Operators which report `PROGRESSING=True` indicate that changes to the managed component are in the
+process of being rolled out. These changes can be new versions of component (in an upgrade), or
+changes applied to the component's desired configuration.
+
+Operators which report `DEGRADED=True` indicate that the managed component is operating at reduced
+level of service than what is desired. This does not mean that the component is out of service -
+however, degraded operators trigger an alert and generally require administrator intervention to
+resolve.
+
+Operators which report `AVAILABLE=False` indicate that the managed component is not operational.
+This means that one or more critical features of OKD will not work as desired. Operators which are
+not available will trigger an alert and generally require administrator intervention to resolve.
+
+If an operator is reporting `AVAILABLE=False` or `DEGRADED=True`, the cluster cannot be
+automatically upgraded.
```

Comment:
i'm not sure if this is actually true.  as far as i know we don't block starting an upgrade because we want you to be able to upgrade your way out of a problem.  it will prevent the upgrade from completing though.

(also not clear what "automatically" means in this sentence).